### PR TITLE
enh(conf): add event_queues_total_size to broker conf (#139)

### PR DESCRIPTION
**Pull request by a-launois**
BACKPORT for #139 + Change sql update script name to from 23.04.0 to 22.10.2

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [x] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software

## Target serie

- [ ] 21.10.x
- [ ] 22.04.x
- [x] 22.10.x
- [ ] 23.04.x (master)

### CODE REVIEW

File: centreon/www/class/centreonConfigCentreonBroker.php [hunk 1]
```diff
@@ -745,6 +746,11 @@ public function insertConfig(array $values): bool
                 (int) $this->checkEventMaxQueueSizeValue($values['event_queue_max_size']),
                 \PDO::PARAM_INT
             );
+            $stmt->bindValue(
+                ':event_queues_total_size',
+                ! empty($values['event_queues_total_size']) ? (int) $values['event_queues_total_size'] : null,
```
Comment (abuathier) [inline centreon/www/class/centreonConfigCentreonBroker.php:751]: Suggestion of "light syntax", but a bit tricky for those who does not understand well the php syntax avec operator precedence.
```suggestion
                (int)($values['event_queues_total_size'] ?? 0) ?: null,
```

File: centreon/www/class/centreonConfigCentreonBroker.php [hunk 2]
```diff
@@ -863,6 +870,11 @@ public function updateConfig(int $id, array $values)
                 (int)$this->checkEventMaxQueueSizeValue($values['event_queue_max_size']),
                 \PDO::PARAM_INT
             );
+            $stmt->bindValue(
+                ':event_queues_total_size',
+                ! empty($values['event_queues_total_size']) ? (int) $values['event_queues_total_size'] : null,
```
Comment (abuathier) [inline centreon/www/class/centreonConfigCentreonBroker.php:875]: Idem + haut
```suggestion
                (int)($values['event_queues_total_size'] ?? 0) ?: null,
```

File: centreon/www/install/php/Update-22.10.1.php [hunk 3]
```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * Copyright 2005 - 2022 Centreon (https://www.centreon.com/)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
```
Comment (abuathier) [inline centreon/www/install/php/Update-22.10.1.php:10]:
```suggestion
 * https://www.apache.org/licenses/LICENSE-2.0
```

File: centreon/www/install/php/Update-22.10.2.php [hunk 4]
```diff
@@ -0,0 +1,46 @@
+<?php
+
+/*
+ * Copyright 2005 - 2022 Centreon (https://www.centreon.com/)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
```
Comment (abuathier) [inline centreon/www/install/php/Update-22.10.2.php:10]:
```suggestion
 * https://www.apache.org/licenses/LICENSE-2.0
```

**Review by abuathier — APPROVED** [review]

File: centreon/www/class/centreonConfigCentreonBroker.php (hunk 1 above)
Comment (a-launois) [reply centreon/www/class/centreonConfigCentreonBroker.php:751]: I take note of this but this PR is a backport, I will keep code unchanged

**Review by wtermellil — APPROVED** [review]

**Pull request merged**